### ISSUE TITLE
Error when diff tool contains a space in the filename

### ISSUE THREAD

**Issue by wesleysmith** [deadlydog/VS.DiffAllFiles#2]
I use a git with a diff tool whose path contains a space.  Here's what's in my .git/config file:

```
[diff]
    tool = winmerge
[difftool "winmerge"]
    cmd = "'C:/Program Files (x86)/WinMerge/WinMergeU.exe'" -e "$LOCAL" "$REMOTE"
```

This works, and I can run "git difftool filename" from the command line, and I can also right click on files from Visual Studio and choose diff and it works as expected.

However, with diff all files, I see this error:

```
Unexpected Error Occurred:
It appears that your diff tool is not configured correctly. Most likely the path to the executable is not correct or is not the correct format (e.g. requires backslashes or double quotes to be escaped). We attempted to invoke the diff tool using the following command: '"'C:\Program Files (x86)\WinMerge\WinMergeU.exe'" -e "C:\Users\xxx\AppData\Local\Temp\DiffAllFilesTemp\einxycfn.nsy\source_CombinedFiles" "C:\Users\xxx\AppData\Local\Temp\DiffAllFilesTemp\einxycfn.nsy\target_CombinedFiles"'.
The system cannot find the file specified
```

I tried moving the executable to a different directory and changing the .git config to this:

```
[diff]
    tool = winmerge
[difftool "winmerge"]
    cmd = "d:/TEMP/WinMerge/WinMergeU.exe" -e "$LOCAL" "$REMOTE"
```

and that worked fine.  Seems like maybe the single quotes around the executable name (which are required to make git work) are messing up diff all files?

**Comment by deadlydog** [comment]
Thanks for reporting this issue. I guess I never had notifications setup on this repo so I didn't see this until now. I'll try and get it fixed up in the next release.

**Comment by deadlydog** [comment]
Hey @wesleysmith, I finally got some time to look into this. The problem was not the space in the file path, but that it was wrapped in single quotes. I've fixed the bug in pull request #11 and it will be available in the next version of the tool (should be published within the next few days).
